pig-gradle-plugin/build.gradle.kts

```diff
@@ -0,0 +1,55 @@
+plugins {
+    id("java-gradle-plugin")
+    id("org.jetbrains.kotlin.jvm") version "1.4.0"
+    id("com.gradle.plugin-publish") version "1.0.0"
+}
+
+repositories {
+    mavenCentral()
+}
+
+version = "0.5.1-SNAPSHOT"
+group = "org.partiql"
+
+dependencies {
+    // It is non-trivial to depend on a local plugin within a gradle project
+    // The simplest way is using a composite build: https://docs.gradle.org/current/userguide/composite_builds.html
+    // Other methods involved adding the build/lib/... jar to classpath, or publish to maven local
+    // By adding the plugin as a dep in `pig-tests`, I cannot use an included build of `pig` in the plugin
+    // Hence it's much simpler to use the latest published version in the plugin
+    implementation("org.partiql:partiql-ir-generator:0.5.0")
```

Comment:
The plugin, runtime, and generator should all be a single gradle project. The issue here is caused by the pig-tests package which, as is, should not be in the main pig gradle project. The pig-tests needs to be fixed to use the generator directly rather than the executable. Executable/plugin usage should be in a separate "pig-example" project which would avoid these oddities. Right now, example usage and testing are intermixed. https://github.com/partiql/partiql-ir-generator/issues/130